test/server/test_command_handler.py

```diff
@@ -35,9 +35,16 @@ def __init__(self, *args, **kwargs):
     def test_command_handler(self):
         transport = mock.Mock()
```

Comment:
Not sure what we are testing here. 

Command handler logic for executing the whole pipeline and this test should be testing if everything happens as expected.